test/test_suite_test.go

```diff
@@ -154,12 +154,12 @@ var _ = BeforeSuite(func() {
 
 		// Name for K8s VMs depends on K8s version that is running.
 
-		err = helpers.CreateVM(helpers.K8s1VMName())
-		if err != nil {
-			Fail(fmt.Sprintf("error starting VM %q: %s", helpers.K8s1VMName(), err))
-		}
-
-		err = helpers.CreateVM(helpers.K8s2VMName())
+		//err = helpers.CreateVM(helpers.K8s1VMName())
```

Comment:
Why you commented this code? It's important to start the proeject on Jenkins

Reply:
Leftover